### PR TITLE
SVG accessibility fixes

### PR DESCRIPTION
**Pull request by patrickhlauke**
- add `focusable="false"` to all SVGs, to stop IE from including the `<svg>` in its default focus cycle
- completely hide decorative SVGs using `aria-hidden="true"`
- add `role="img"` to meaningful/"content" SVGs, ensure they have an appropriate `<title>` and/or `aria-label`

### CODE REVIEW

**Comment by patrickhlauke** [comment]
@XhmikosR force-pushed a change, and expanded this PR quite a bit to remove some redundant/overly descriptive text (now that this is constructed from both title and text). also includes changes to SVGO configuration to not remove `role` and to add/force `focusable="false"`

File: build/svgo.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ js2svg:
   indent: 2
 
 plugins:
+  - addAttributesToSVGElement:
+      attributes:
+        - focusable: false
+        - data-image: icon
```
Comment (XhmikosR) [inline build/svgo.yml]: Why do we need `data-image`?

Also, note that we'd need to run svgo with the updated config before merging this.

Comment (patrickhlauke) [reply build/svgo.yml]: sorry, cargo-culted that `data-image` part from the plugin's documentation, but you're quite right it's not needed. force-pushed an updated version without it.

when you say "we'd need to run svgo" ... where do you mean? against the files in docs? when i initially tried, it choked on the `{{ ... }}` etc liquid template bits.

Comment (XhmikosR) [reply build/svgo.yml]: Yeah, it won't work for those where we have liquid code, but it'll work for the rest. I will take care of it and push here later.

Comment (XhmikosR) [reply build/svgo.yml]: Actually I looked at it and it's OK as it is since svgo adds whitespace with this config.

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**